Dockerfile

```diff
@@ -92,6 +92,11 @@ RUN cget -p $PREFIX install -f /dev-requirements.txt -DMIOPEN_CACHE_DIR=""
 
 RUN pip3 install onnx==1.7.0 numpy==1.18.5 typing==3.7.4 pytest==6.0.1
 
+# Download real models to run onnx unit tests
+ENV ONNX_HOME=$HOME
```

Comment:
Is it possible to set this in the python script or cmake script(I think you can set the `ENVIRONMENT` test property)?